docs/docs/reaching-out-to-user.mdx

```diff
@@ -0,0 +1,459 @@
+            kill_on_user_message=False,
+        )
+
+        return [reminder]
+```
+Note that this action requires the `datetime` and `rasa_sdk.events` packages.
+
+The first argument for the `ReminderScheduled` event is the reminder's name, in this case, `EXTERNAL_reminder`.
+The reminder name will be used later as an intent to trigger a reaction to the reminder.
+As with external events, it's recommended to name the reminder name with the
+`EXTERNAL_` prefix to make it easier to see what's going on in your training data.
+
+You can see that the last messages' `entities` are also passed to the reminder.
+This allows the action that reacts to the reminder to make use of the entities
+from the user's scheduling message.
+
+For example, if you want your assistant to remind you to call a friend, you could
+send it a message like "Remind me to call Paul". If `Paul` is extracted as a `PERSON`
+entity, the action reacting to the reminder can use it to say "Remember to call Paul!"
+
+### Reacting to Reminders
+
+As with external events, the bot reaches out to the user after receiving a
+POST request to the `trigger_intent` endpoint. Reminders, however, send
+the request to the right conversation ID automatically after a certain amount of time using
+the name that you define in the `ReminderScheduled` event.
+
+To define a reaction to the reminder, you only need to write a [rule](#rules) that
+tells the bot what action to take when it receives the reminder intent.
+
+In the call reminder example, you want to use the entities that come with the
+reminder to be reminded to call specific people, so you need to write a custom
+action that does that:
+
+```python
+class ActionReactToReminder(Action):
+    """Reminds the user to call someone."""
+
+    def name(self) -> Text:
+        return "action_react_to_reminder"
+
+    async def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[Dict[Text, Any]]:
+
+        name = next(tracker.get_latest_entity_values("PERSON"), "someone")
+        dispatcher.utter_message(f"Remember to call {name}!")
+
+        return []
+```
+
+### Cancelling Reminders
+
+To cancel a reminder that you've already scheduled, you need a custom action
+that returns the `ReminderCancelled()` event.
+
+Returning `ReminderCancelled()` cancels all the reminders that are currently scheduled.
+If you only want to cancel certain reminders, you can specify some parameters by which to narrow down the scheduled reminders:
+
+* `ReminderCancelled(intent="greet")` cancels all reminders with intent `greet`
+
+* `ReminderCancelled(entities={})` cancels all reminders with the given entities
+
+* `ReminderCancelled("...")` cancels the one unique reminder with the given name “`...`” that you supplied
+  during its creation
+
+
+For the call reminder example, you can define a custom action `action_forget_reminders` that cancels
+all reminders:
```

Comment:
ok and how does it get triggered? I assume by the user saying "cancel reminder"? Then we should have a rule/story for this

Reply:
oh apparently that's defined in the section below, i feel like that may confuse people?